src/compiler/utilities.ts

```diff
@@ -3793,12 +3786,9 @@ namespace ts {
     }
 
     export function textSpanIntersection(span1: TextSpan, span2: TextSpan) {
-        const intersectStart = Math.max(span1.start, span2.start);
-        const intersectEnd = Math.min(textSpanEnd(span1), textSpanEnd(span2));
-        if (intersectStart <= intersectEnd) {
-            return createTextSpanFromBounds(intersectStart, intersectEnd);
-        }
-        return undefined;

```

Comment:
You haven't done any logic changes here. From what I can see, renamed the variables and re written the logic using ternary operator. Is there anything am missing here?

Reply:
Yeah, this PR isn't meant to change the return values of anything.